docs/Configuration.md

```diff
@@ -4,18 +4,105 @@ title: Configuration
+
+{: #code-workspace-simple-debug-process}
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "objectscript",
+            "request": "attach",
+            "name": "Example-attach-to-process", 
+            "processId": "${command:PickProcess}"
+        }
+    ]
+}
+```
+
+Note that `"configurations"` is an array, so you can have multiple configurations and choose the one to use from a dropdown menu in the Debug pane.
+
 {: #code-workspaces}
-## VS Code Workspaces 
 
-To work with VS Code, you need to open a workspace. In the simplest setup, a VS Code workspace is just the root folder of your project. Workspace settings and task configurations are stored in the root folder in the `settings.json` file in a folder called `.vscode`. Debugging launch configurations are stored in `launch.json`, also in `.vscode`.
+## VS Code Workspaces
 
-If you need to have more than one root folder in a VS Code workspace, use a feature called multi-root workspaces. See [Multi-root Workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) in the VS Code documentation.
+If your project requires more than a single root folder, you need to use a feature called multi-root workspaces. See [Multi-root Workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) in the VS Code documentation.
 
-A multi-root workspace is defined by a `*.code-workspace` file. The file can have any name followed by *.code-workspace*, for example `test.code-workspace`. The `*.code-workspace` file stores information about what folders are in the workspace, and may also store other settings that would otherwise be stored in the settings.json or launch.json files. Settings in a folder's `.vscode/settings.json` or `.vscode/launch.json` will override those in the `*.code-workspace` file, so be careful to use one or the other unless you truly need to leverage this flexibility. You can have a workspace file even if you are only working with a single root folder. Indeed, if you are [working server-side](../serverside/) you will always be using a workspace file.
+Since you have more than one root folder, settings are stored in a file with a `*.code-workspace` suffix. The file can have any name followed by *.code-workspace*, for example `test.code-workspace`. This file can be located anywhere. The `*.code-workspace` file stores information about what folders are in the workspace, and may also store other settings that would otherwise be stored in `settings.json` or `launch.json`. Settings in a folder's `.vscode/settings.json` or `.vscode/launch.json` will override those in the `*.code-workspace` file, so be careful to use one or the other unless you truly need to leverage this flexibility. You can have a workspace file even if you are only working with a single root folder. Indeed, if you are [working server-side](../serverside/) you will always be using a workspace file.
```

Comment:
```suggestion
In this case settings are stored in a file with a `*.code-workspace` suffix. The filename's extension must be *.code-workspace*, for example `test.code-workspace`. This workspace file can be located anywhere. It defines what root folders the workspace consists of, and may also store other settings that would otherwise be stored in `settings.json` or `launch.json`. Settings in a root folder's `.vscode/settings.json` or `.vscode/launch.json` will override those in the workspace file, so be careful to use one or the other unless you truly need to leverage this flexibility.

You can have a workspace file even if you are only working with a single root folder. Indeed, if you are [working server-side](../serverside/) you will always be using a workspace file.
```